test/fixtures/poro.rb

```diff
@@ -309,11 +309,27 @@ class CachingCommentSerializer < CommentSerializer
     cache expires_in: 1.day, skip_digest: true
   end
 
-  class CachingModelSerializer < ModelSerializer
+  class CachingModelSerializer < ActiveModel::Serializer
```

Comment:
Sorry, meant diff as suggesting interface change rather than a working didd

On Tue, Apr 19, 2016 at 10:22 PM Lucas Hosseini <notifications@github.com>
wrote:

> In test/fixtures/poro.rb
> <https://github.com/rails-api/active_model_serializers/pull/1690#discussion_r60345797>
> :
>
> > @@ -309,11 +309,27 @@ class CachingCommentSerializer < CommentSerializer
> >      cache expires_in: 1.day, skip_digest: true
> >    end
> >
> > -  class CachingModelSerializer < ModelSerializer
> > +  class CachingModelSerializer < ActiveModel::Serializer
>
> Hmm it's an array though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1690/files/b3518271c473ef14e73463cd5e93897a3d8e3785..68715b8f99bc29677e8a47bb3f305f23c077024b#r60345797>
>